Description: There is a package for assets, let's avoid confusion in the Online Updater
Index: RetroArch-git2/menu/menu_displaylist.c
===================================================================
--- RetroArch-git2.orig/menu/menu_displaylist.c
+++ RetroArch-git2/menu/menu_displaylist.c
@@ -3082,12 +3082,14 @@ static unsigned menu_displaylist_parse_o
    }
 
 #ifdef HAVE_UPDATE_ASSETS
+/*
    menu_entries_append_enum(info->list,
          msg_hash_to_str(MENU_ENUM_LABEL_VALUE_UPDATE_ASSETS),
          msg_hash_to_str(MENU_ENUM_LABEL_UPDATE_ASSETS),
          MENU_ENUM_LABEL_UPDATE_ASSETS,
          MENU_SETTING_ACTION, 0, 0);
    count++;
+*/
 #endif
 
    menu_entries_append_enum(info->list,
